### PR TITLE
feat(kskeleton) override transparency

### PR DESCRIPTION
**Pull request by jackkav**
### Summary

The delay-milliseconds implementation currently handles the skeleton flashing by conditionally rendering after a timeout.
I propose to change this in favour of rendering transparently until after the timeout.

This way I do not need to approximate the pixel height of the skeleton and presented data to avoid components that flow vertically below from jumping up and down.

#### Changes made:
* added a css class
* moved isVisible condition to css class
* added example in docs to demonstrate the problem
* added a fade in

### PR Checklist
- [x] Does not introduce dependencies
- [x] **Functional:** all changes do not break existing APIs and if so, bump major version.
- [x] **Tests pass:** check the output of yarn test packages/<Kongponent>
- [x] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
- [x] **Framework style:** abides by the essential rules in Vue's style guide
- [x] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs), or leftover comments
- [x] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

**Comment by jackkav** [comment]
I've tested this with the reporting skeletons and the cards in the docs, could use some help to find if there are other cases that could be undesirably affected.

File: packages/KSkeleton/KSkeleton.vue [hunk 1]
```diff
@@ -104,3 +103,8 @@ export default {
   }
 }
 </script>
+<style>
+.transparent .box {
+  background: transparent;
```
Comment (darrenjennings) [inline packages/KSkeleton/KSkeleton.vue:108]: can you rename class to `bg-transparent`, add it to _utils.scss and remove this style block?

**Review by darrenjennings — CHANGES_REQUESTED** [review]

File: packages/KSkeleton/KSkeleton.vue (hunk 1 above)
Comment (darrenjennings) [reply packages/KSkeleton/KSkeleton.vue:108]: bonus points for animating their entry into the dom 😎

Comment (jackkav) [reply packages/KSkeleton/KSkeleton.vue:108]: I'm not sure I can remove this style blocks as its needs the grandchild selector `.box` in order to alter the skeleton sub-sub-component skeletonbox styling. I may be mistaken.

**Review by darrenjennings — APPROVED** [review]

**Pull request merged**